features/netsocket/OnboardNetworkStack.h

```diff
@@ -39,6 +39,8 @@ class OnboardNetworkStack : public NetworkStack {
      */
     static OnboardNetworkStack &get_default_instance();
 
+    virtual OnboardNetworkStack *onboardNetworkStack() { return this; }
```

Comment:
See below (or above, depending on GitHub ordering). I don't think we need this on this pr